websites/D/Discord.style/presence.ts

```diff
@@ -4,23 +4,37 @@ const presence = new Presence({
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
 presence.on("UpdateData", async () => {
-	const showTimestamp = await presence.getSetting<boolean>("timestamp"),
+	let presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/1wKlRwf.png",
+	};
+
+	const { href, pathname } = document.location,
+		showTimestamp = await presence.getSetting<boolean>("timestamp"),
 		showButtons = await presence.getSetting<boolean>("buttons"),
-		presenceData: PresenceData = {
-			largeImageKey: "discordstyle_logo",
+		pages: Record<string, PresenceData> = {
+			"/bot": {
+				details: "Viewing 🤖 bot page",
+				buttons: [{ label: "View Page", url: href }],
+			},
+			"/search": {
+				details: "Viewing 🔎 search page",
+				buttons: [{ label: "View Page", url: href }],
```

Comment:
no don't do that

Reply:
Agreed, arbitrary buttons are not allowed

Reply:
Alright, will change.